workshop/Lab3/README.md

```diff
@@ -1,45 +1,47 @@
-# Lab 3: Scale and update apps natively, building multi-tier applications.
+# Lab 3: Scale and update apps natively by building multi-tier applications
 
-In this lab you'll learn how to deploy the same guestbook application we
-deployed in the previous labs, however, instead of using the `kubectl`
-command line helper functions we'll be deploying the application using
+In this lab, you'll deploy the same guestbook application from the 
+previous lab by using a different method. Instead of using `kubectl`,
+you will deploy the application by using
 configuration files. The configuration file mechanism allows you to have more
 fine-grained control over all of resources being created within the
 Kubernetes cluster.
 
-Before we work with the application we need to clone a github repo:
+Before we work with the application, we need to clone a github repo:
 
 ```
 $ git clone https://github.com/IBM/guestbook.git
 ```
 
 This repo contains multiple versions of the guestbook application
-as well as the configuration files we'll use to deploy the pieces of the application.
+as well as the configuration files that we'll use to deploy the pieces of the application.
 
-Change directory by running the command `cd guestbook`. You will find all the
+Change the directory by running the command `cd guestbook`. You will find all the
 configurations files for this exercise under the directory `v1`.
 
-# 1. Scale apps natively
+## 1. Scale apps natively
 
-Kubernetes can deploy an individual pod to run an application but when you
+Kubernetes can deploy an individual pod to run an application, but when you
 need to scale it to handle a large number of requests a `Deployment` is the
 resource you want to use.
-A Deployment manages a collection of similar pods. When you ask for a specific number of replicas
-the Kubernetes Deployment Controller will attempt to maintain that number of replicas at all times.
+A Deployment manages a collection of similar pods. When you ask for a specific number of replicas,
+the Kubernetes Deployment Controller will attempt to maintain that replica number at all times.
 
-Every Kubernetes object we create should provide two nested object fields
-that govern the object’s configuration: the object `spec` and the object
+### Create objects
+
+Every Kubernetes object that we create should provide two nested object fields
```

Comment:
I didn't notice this before, but this sentence is a bit misleading since the user doesn't normally provide the "status" part, as mentioned below the Kube system will normally do it. So perhaps we just need to say "Every Kubernetes object that is created will have two nested object fields: `spec` and `status`. `spec` defines the desired state (as specified by the person managing the object), and the `status` contains the information about ...

Reply:
Done.